### DISCUSSION TITLE
Slack notifications - webhook payload?

### DISCUSSION THREAD

**Discussion by mdzidic** [hyperjumptech/monika#378] [Ideas]
Hello,

Can you expand Slack notifications with other Slack webhook payload parameters?

Example:
```
 {
   "id": "unique-id-slack",
   "type": "slack",
   "data": {
     "url": "https://YOUR_SLACK_INCOMING_WEBHOOK_URL",
     "channel": "#YOUR_SLACK_CHANNEL",
     "username": "YOUR_BOT_NAME"
   }
 }
```

**Comment by nicnocquee** [comment]
Can you point to a documentation for other webhook payload parameters? CMIIW, but in this [doc here,](https://api.slack.com/messaging/webhooks#create_a_webhook) it says

```
 You cannot override the default channel (chosen by the user who installed your app), username, or icon when you're using Incoming Webhooks to post messages. Instead, these values will always inherit from the associated Slack app configuration.
```

**Reply by mdzidic** [reply]
It seems that Slack is trying to deprecate "Incoming WebHooks" where all of these options are still working... 😞 

Slack Notice:
> Please note, this is a legacy custom integration - an outdated way for teams to integrate with Slack. These integrations lack newer features and they will be deprecated and possibly removed in the future. We do not recommend their use. Instead, we suggest that you check out their replacement: Slack apps.

Anyway, documentation under Incoming WebHooks configuration says:
> Incoming webhooks have a default channel, but it can be overridden in your JSON payload. A public channel can be specified with "channel": "#other-channel", and a Direct Message with "channel": "@username".
